### PR TITLE
Fixed broken links to our Patch Upgrade doc

### PR DESCRIPTION
**Pull request by jwitz**
When we cut 0.23, I renamed the file name for our [Patch Upgrade doc](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-patch) do more closely reflect its content. I hadn't considered that this would affect our links to the file in other places in docs. 

To fix this, I've updated the links in 0.23 and 0.16 docsets to point to the correct file name. I didn't rename the file in earlier versions, so those links are unchanged.

### CODE REVIEW

File: enterprise/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ There are many tools at your disposal for administrating Astronomer:
 * [Using Kibana](https://www.astronomer.io/docs/enterprise/stable/monitor/kibana-logging/)
 * [Using kubectl](https://www.astronomer.io/docs/enterprise/stable/troubleshoot/kubectl/)
 * [Pulling Postgres Credentials](https://www.astronomer.io/docs/enterprise/stable/customize-airflow/access-airflow-database/)
-* [Upgrading](https://www.astronomer.io/docs/enterprise/stable/manage-astronomer/upgrade-astronomer/)
+* [Upgrading](https://www.astronomer.io/docs/enterprise/stable/manage-astronomer/upgrade-patch/)
```
Comment (paolaperaza) [inline enterprise/README.md:45]:
```suggestion
* [Upgrade to a Patch Version of Astronomer Enterprise](https://www.astronomer.io/docs/enterprise/stable/manage-astronomer/upgrade-patch/)
```

File: enterprise/next/09_resources/03_version-compatibility-reference.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ It's worth noting that while the tables below reference the minimum compatible v
...
 | v0.16                | 1.16, 1.17, 1.18 | 3    | 0.12, 0.13.5      | 9.6+     | 1.10.5, 1.10.7, 1.10.10, 1.10.12, 1.10.14        | 3.6, 3.7, 3.8 |
-| v0.23 (*Coming Soon*)| 1.16, 1.17, 1.18 | 3    | 0.13.5      | 9.6+     | 1.10.5, 1.10.7, 1.10.10, 1.10.12, 1.10.14, 2.0.0 | 3.6, 3.7, 3.8 |
+| v0.23                | 1.16, 1.17, 1.18 | 3    | 0.13.5      | 9.6+     | 1.10.5, 1.10.7, 1.10.10, 1.10.12, 1.10.14, 2.0.0 | 3.6, 3.7, 3.8 |
 
-> **Note:** Astronomer v0.16.9+ is required to run Astronomer Certified 1.10.12, and Astronomer v0.16.15+ is required to run Astronomer Certified 1.10.14. For instructions on how to upgrade to the latest version of Astronomer, refer to our ["Enterprise Upgrade Guide"](https://www.astronomer.io/docs/enterprise/next/manage-astronomer/upgrade-astronomer/).
+> **Note:** Astronomer v0.16.9+ is required to run Astronomer Certified 1.10.12, and Astronomer v0.16.15+ is required to run Astronomer Certified 1.10.14. For instructions on how to upgrade to the latest version of Astronomer, read [Upgrade to Astronomer Enterprise v0.23](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-to-0-23/).
```
Comment (paolaperaza) [inline enterprise/next/09_resources/03_version-compatibility-reference.md:20]:
```suggestion
> **Note:** Astronomer v0.16.9+ is required to run Astronomer Certified 1.10.12, and Astronomer v0.16.15+ is required to run Astronomer Certified 1.10.14. For instructions on how to upgrade to an Astronomer v0.16 patch version, read [Upgrade to a Patch Version of Astronomer](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-patch).
```

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ $ kubectl apply -f https://raw.githubusercontent.com/astronomer/astronomer/maste
 While your platform is upgrading, monitor your pods to ensure that no errors occur. To do so, first find the names of your pods by running the following command:
 
 ```sh
-$ kubectl get pods | grep upgrade-astronomer
+$ kubectl get pods | grep upgrade-patch
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:82]:
```suggestion
$ kubectl get pods | grep upgrade-astronomer
```

@jwitz I don't think this was an intended change?

File: enterprise/v0.16/06_manage-astronomer/07_apply-platform-config.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ For example, you can:
 
 To configure these settings, follow the steps below.
 
-> **Note:** If you're interested in upgrading Astronomer to a new version of the platform, read [Upgrade Astronomer](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-astronomer).
+> **Note:** If you're interested in upgrading Astronomer to the latest version of the platform, read [Upgrade to Astronomer Enterprise v0.23](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-to-0-23).
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/07_apply-platform-config.md:20]:
```suggestion
> **Note:** If you're interested in upgrading Astronomer to the latest v0.16 patch version, read [Upgrade to a Patch Version of Astronomer](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-patch). For instructions on how to upgrade to the latest minor version of the platform, read [Upgrade to Astronomer Enterprise v0.23](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-to-0-23). 
```

File: enterprise/v0.16/09_resources/01_release-notes.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ description: "Astronomer Enterprise release notes."
 
 As of Astronomer v0.16, Astronomer releases will be made generally available to Enterprise customers on a quarterly basis as part of a long-term support (LTS) release model. Critical security and bug fixes will be regularly shipped as patch versions that follow LTS releases. Patch releases will be made available between quarterly LTS releases and require a simple upgrade process.
 
-For instructions on how to upgrade to a patch version, refer to ["Upgrade Astronomer"](/docs/enterprise/v0.16/manage-astronomer/upgrade-astronomer/).
+For instructions on how to upgrade to a patch version, read [Upgrade to a Patch Version](/docs/enterprise/v0.16/manage-astronomer/upgrade-patch/).
```
Comment (paolaperaza) [inline enterprise/v0.16/09_resources/01_release-notes.md:11]:
```suggestion
For instructions on how to upgrade to the latest v0.16 patch version of Astronomer, read [Upgrade to a Patch Version of Astronomer](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-patch). If you're interested in upgrading from Astronomer v0.16.x to the latest minor version of the platform, read [Upgrade to Astronomer Enterprise v0.23](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-to-0-23).
```

File: enterprise/v0.16/09_resources/03_version-compatibility-reference.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ It's worth noting that while the tables below reference the minimum compatible v
 | v0.16                | 1.16, 1.17, 1.18 | 3    | 0.12, 0.13.5      | 9.6+     | 1.10.5, 1.10.7, 1.10.10, 1.10.12, 1.10.14        | 3.6, 3.7, 3.8 |
 | v0.23 (*Coming Soon*)| 1.16, 1.17, 1.18 | 3    | 0.13.5      | 9.6+     | 1.10.5, 1.10.7, 1.10.10, 1.10.12, 1.10.14, 2.0.0 | 3.6, 3.7, 3.8 |
 
-> **Note:** Astronomer v0.16.9+ is required to run Astronomer Certified 1.10.12, and Astronomer v0.16.15+ is required to run Astronomer Certified 1.10.14. For instructions on how to upgrade to the latest version of Astronomer, refer to our ["Enterprise Upgrade Guide"](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-astronomer/).
+> **Note:** Astronomer v0.16.9+ is required to run Astronomer Certified 1.10.12, and Astronomer v0.16.15+ is required to run Astronomer Certified 1.10.14. For instructions on how to upgrade to the latest version of Astronomer, read [Upgrade to Astronomer Enterprise v0.23](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-to-0-23/).
```
Comment (paolaperaza) [inline enterprise/v0.16/09_resources/03_version-compatibility-reference.md:20]:
```suggestion
> **Note:** Astronomer v0.16.9+ is required to run Astronomer Certified 1.10.12, and Astronomer v0.16.15+ is required to run Astronomer Certified 1.10.14. For instructions on how to upgrade, read [Upgrade to a Patch Version of Astronomer](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-patch).
```

File: enterprise/v0.23/09_resources/01_release-notes.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ description: "Astronomer Enterprise release notes."
 
 As of Astronomer v0.16, Astronomer releases will be made generally available to Enterprise customers on a quarterly basis as part of a long-term support (LTS) release model. Critical security and bug fixes will be regularly shipped as patch versions that follow LTS releases. Patch releases will be made available between quarterly LTS releases and require a simple upgrade process.
 
-For instructions on how to upgrade to a patch version, refer to ["Upgrade Astronomer"](/docs/enterprise/v0.23/manage-astronomer/upgrade-astronomer/).
+For instructions on how to upgrade to a patch version, read [Upgrade to a Patch Version](/docs/enterprise/v0.23/manage-astronomer/upgrade-patch/).
```
Comment (paolaperaza) [inline enterprise/v0.23/09_resources/01_release-notes.md:11]:
```suggestion
For instructions on how to upgrade to an Astronomer v0.23 patch version, read [Upgrade to a Patch Version](/docs/enterprise/v0.23/manage-astronomer/upgrade-patch/).
```

File: enterprise/v0.23/06_manage-astronomer/06_apply-platform-config.md [hunk 8]
```diff
@@ -17,7 +17,7 @@ For example, you can:
 
 To configure these settings, follow the steps below.
 
-> **Note:** If you're interested in upgrading Astronomer to a new version of the platform, read [Upgrade Astronomer](https://www.astronomer.io/docs/enterprise/v0.23/manage-astronomer/upgrade-astronomer).
+> **Note:** If you're interested in upgrading Astronomer to a new version of the platform, read [Upgrade to a Patch Version](https://www.astronomer.io/docs/enterprise/v0.23/manage-astronomer/upgrade-patch).
```
Comment (paolaperaza) [inline enterprise/v0.23/06_manage-astronomer/06_apply-platform-config.md:20]:
```suggestion
> **Note:** If you're interested in upgrading to the latest Astronomer v0.23 patch version, read [Upgrade to a Patch Version of Astronomer](https://www.astronomer.io/docs/enterprise/v0.23/manage-astronomer/upgrade-patch).
```

File: enterprise/v0.23/09_resources/03_version-compatibility-reference.md [hunk 9]
```diff
@@ -15,9 +15,9 @@ It's worth noting that while the tables below reference the minimum compatible v
...
 | v0.16                | 1.16, 1.17, 1.18 | 3    | 0.12, 0.13.5      | 9.6+     | 1.10.5, 1.10.7, 1.10.10, 1.10.12, 1.10.14        | 3.6, 3.7, 3.8 |
-| v0.23 (*Coming Soon*)| 1.16, 1.17, 1.18 | 3    | 0.13.5      | 9.6+     | 1.10.5, 1.10.7, 1.10.10, 1.10.12, 1.10.14, 2.0.0 | 3.6, 3.7, 3.8 |
+| v0.23                | 1.16, 1.17, 1.18 | 3    | 0.13.5      | 9.6+     | 1.10.5, 1.10.7, 1.10.10, 1.10.12, 1.10.14, 2.0.0 | 3.6, 3.7, 3.8 |
 
-> **Note:** Astronomer v0.16.9+ is required to run Astronomer Certified 1.10.12, and Astronomer v0.16.15+ is required to run Astronomer Certified 1.10.14. For instructions on how to upgrade to the latest version of Astronomer, refer to our ["Enterprise Upgrade Guide"](https://www.astronomer.io/docs/enterprise/next/manage-astronomer/upgrade-astronomer/).
+> **Note:** Astronomer v0.16.9+ is required to run Astronomer Certified 1.10.12, and Astronomer v0.16.15+ is required to run Astronomer Certified 1.10.14. For instructions on how to upgrade to the latest version of Astronomer, refer to our ["Enterprise Upgrade Guide"](https://www.astronomer.io/docs/enterprise/next/manage-astronomer/upgrade-patch/).
```
Comment (paolaperaza) [inline enterprise/v0.23/09_resources/03_version-compatibility-reference.md:20]:
```suggestion
> **Note:** Astronomer v0.16.9+ is required to run Astronomer Certified 1.10.12, and Astronomer v0.16.15+ is required to run Astronomer Certified 1.10.14. For instructions on how to upgrade to an Astronomer v0.16 patch version, read [Upgrade to a Patch Version of Astronomer](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/upgrade-patch).
```

**Review by paolaperaza — CHANGES_REQUESTED** [review]

File: enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md (hunk 3 above)
Comment (jwitz) [reply enterprise/v0.16/06_manage-astronomer/06_upgrade-to-0-23.md:82]: Shoot, didn't anticipate that the file name would be the same as something that's actually in the system!

**Comment by jwitz** [comment]
Thanks for these suggestions @paolaperaza , I agree with all of them. For the next release, we might consider ways to future-proof these links so that we don't have to update them every time. For now though, I'm adding these doc sections to [Notion](https://www.notion.so/astronomerio/Enterprise-New-Version-Checklist-180716f00e854670ad5a45c9bd000032) as bits to look out for when cutting a new Enterprise docset.

**Review by paolaperaza — APPROVED** [review]

**Comment by paolaperaza** [comment]
Sweet, yep this looks good - too many links 💣  Thanks though, @jwitz ! Merging.

**Pull request merged**